### ISSUE TITLE
Exam page has lag when I creating an exam

### ISSUE THREAD

**Issue by mrpau-julius** [learningequality/kolibri#4159]
### Observed behavior
I observed the exam page was lag when I click the checkbox. When I click Stop it, theirs error message in console. 
Refer to Gherkin scenarios :
https://github.com/learningequality/kolibri/blob/gherkin-features/integration_testing/features/coach/coach-create-exams.feature
![lag-error](https://user-images.githubusercontent.com/38421180/44073483-f9afd3ec-9f48-11e8-9762-1e725ec912c6.png)

![error-msg](https://user-images.githubusercontent.com/38421180/44073489-ff5c4cda-9f48-11e8-9eb5-09d79316c194.png)

### Expected behavior

no lag when clicking the checkbox.

### Errors and logs
```
Error: Script terminated by timeout at:
get@http://127.0.0.1:8080/static/default_frontend/default_frontend-0.11.0a5.js:7:6674
exerciseIsSelected/<@http://127.0.0.1:8080/static/coach_module/coach_module-0.11.0a5.js:1:61791
exerciseIsSelected@http://127.0.0.1:8080/static/coach_module/coach_module-0.11.0a5.js:1:61751
allExercisesWithinTopicSelected@http://127.0.0.1:8080/static/coach_module/coach_module-0.11.0a5.js:1:61374
kolibriGlobal</</Le.prototype.get@http://127.0.0.1:8080/static/default_frontend/default_frontend-0.11.0a5.js:7:15849
kolibriGlobal</</Le.prototype.evaluate@http://127.0.0.1:8080/static/default_frontend/default_frontend-0.11.0a5.js:7:17006
$e/<@http://127.0.0.1:8080/static/default_frontend/default_frontend-0.11.0a5.js:7:18899
$e<@http://127.0.0.1:8080/static/coach_module/coach_module-0.11.0a5.js:1:62248
kolibriGlobal</</</t.prototype._render@http://127.0.0.1:8080/static/default_frontend/default_frontend-0.11.0a5.js:7:32600
kolibriGlobal</</fn.prototype.$mount/</<@http://127.0.0.1:8080/static/default_frontend/default_frontend-0.11.0a5.js:7:60780
kolibriGlobal</</Le.prototype.get@http://127.0.0.1:8080/static/default_frontend/default_frontend-0.11.0a5.js:7:15849
kolibriGlobal</</Le.prototype.run@http://127.0.0.1:8080/static/default_frontend/default_frontend-0.11.0a5.js:7:16740
Ae@http://127.0.0.1:8080/static/default_frontend/default_frontend-0.11.0a5.js:7:14878
Zt/<@http://127.0.0.1:8080/static/default_frontend/default_frontend-0.11.0a5.js:7:11350
Wt@http://127.0.0.1:8080/static/default_frontend/default_frontend-0.11.0a5.js:7:10852
kolibriGlobal</t.exports@http://127.0.0.1:8080/static/default_frontend/default_frontend-0.11.0a5.js:91:13652
@http://127.0.0.1:8080/static/default_frontend/default_frontend-0.11.0a5.js:91:10695
y@http://127.0.0.1:8080/static/default_frontend/default_frontend-0.11.0a5.js:91:10553
b@http://127.0.0.1:8080/static/default_frontend/default_frontend-0.11.0a5.js:91:10573
default_frontend-0.11.
```

### Context

Kolibri 0.11.0a5
windows 7 (64-bit)
Firefox 61.0.2 (64-bit)

**Comment by indirectlylit** [comment]
work-arounds based on meeting today with @khangmach @rtibbles @jtamiace 

* hide checkboxes at channel level
* make topics and channel text into purple links
* completely hide rows that are currently disabled

Longer-term (e.g. 0.11.0) we may want to also add validation which shows a modal warning if more than e.g. 200 exercises are selected

**Comment by rtibbles** [comment]
The 0.10.2 fixes for this have been merged, punting further work to 0.11.

**Comment by rtibbles** [comment]
@mrpau-julius could you take a look at the experience you get with this in the current 0.11.0-betas?

**Comment by mrpau-julius** [comment]
@rtibbles sure, I will test it today.

**Comment by mrpau-julius** [comment]
@rtibbles still exists on 0.11.0b2, but different error script.
[kolibri.log](https://github.com/learningequality/kolibri/files/2509847/kolibri.log)
![lag](https://user-images.githubusercontent.com/38421180/47420903-5a6a7e80-d734-11e8-8c23-b648e536979c.png)

**Comment by benjaoming** [comment]
Feedback from community forums seem confirm the issue:

http://community.learningequality.org/t/api-requests-slow-when-creating-an-exam/857/5

> We appear to be having a similar issue but quite a bit worse. When we click on “New Exam” it never gets to the exam creation page. We have waited about 15 minutes and then cancel. I have tried this on two different servers now with the same result. So we have never actually seen the exam screen. Here is our version info:

```
Version: 0.11.0
OS: Windows-8-6.2.9200
Installer: Windows
Server: Kolibri internal server
Database: C:\Users\jronning.kolibri\db.sqlite3
Device name: VM-KOLIBRI
Free disk space: 133 GB
Server time: Sat Dec 08 2018 12:09:29 GMT+0800 (Singapore Standard Time)
Server timezone: Asia/Singapore
```

**Comment by benjaoming** [comment]
Also ref'ing #4110 in case this behavior can be addressed via regression test.

**Comment by indirectlylit** [comment]
I think this is now fixed in 0.12.x. Please re-open if it is not the case
